cmd/node/config/config.toml

```diff
@@ -10,6 +10,9 @@
    # StatusPollingIntervalSec represents the no of seconds between multiple polling for the status for AppStatusHandler
    StatusPollingIntervalSec = 10
 
+   # NodeDisplayName represents the friendly name a user can pick for his node in the status monitor
+   NodeDisplayName = "undefined"
```

Comment:
Might leave it blank as to not have 100 nodes with undefined

Reply:
done